docs/02.API-REFERENCE.md

```diff
@@ -523,6 +544,40 @@ main (void)
 - [jerry_cleanup](#jerry_cleanup)
 
 
+## jerry_get_memory_stats
+
+**Summary**
+
+Get heap memory stats.
+
+**Prototype**
+
+```c
+bool
+jerry_get_memory_stats (jerry_heap_stats_t *out_stats_p);
+```
+
+- `out_stats_p` - out parameter, that gives the stats of the heap.
+- return value
+  - true, if run was successful
+  - false, otherwise. Usually it is because the MEM_STATS feature is not enabled.
+
+**Example**
+
+```c
+jerry_init (JERRY_INIT_EMPTY | JERRY_INIT_MEM_STATS);
```

Comment:
I was trying to find how `JERRY_INIT_MEM_STATS` affects the engine but grepping for this define didn't lead to much insight. Can someone point me to how this flag is actually being consumed?